pkg/cmd/cli/describe/printer.go

```diff
@@ -80,7 +82,41 @@ func NewHumanReadablePrinter(noHeaders bool) *kctl.HumanReadablePrinter {
 	return p
 }
 
-func printParameters(t *templateapi.Template, w io.Writer) error {
+const templateDescriptionLen = 80
```

Comment:
since you've got other stuff on the same line, this is still going to wrap o nan 80 char display...  does it make sense to either make this shorter, or unlimited?  or you just wanted to cap it at some arbitrary length?